definitions/infrastructure_as_a_service.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Infrastructure as a Service (IaaS)
+status: Completed
+category: Concept
+---
+
+## Infrastructure as a Service
+
+### What it is
+
+Infrastructure as a service i.e. IaaS is a [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) service model that offers [virtualized](https://github.com/cncf/glossary/blob/main/definitions/virtualization.md) compute, storage and network resources on-demand over the internet to consumers on a pay-as-you-go cost model.
+In IaaS model, cloud providers owns and operates the hardware and software, it is available in Public cloud, Private cloud and Hybrid cloud deployment options. 
+
+Comparison table
+
+| **On-Premise**       | **IaaS**             | **PaaS**             | **SaaS**             |
+|:------------------|:------------------|:------------------|:------------------|
+| Application      | Application      | Application      | ***Application***      |
+| Data             | Data             | Data             | ***Data***             |
+| Runtime          | Runtime          | ***Runtime***          | ***Runtime***          |
+| Middleware       | Middleware       | ***Middleware***       | ***Middleware***       |
+| Operating System | Operating System | ***Operating System*** | ***Operating System*** |
+| Virtualization   | ***Virtualization***   | ***Virtualization***   | ***Virtualization***   |
+| Server           | ***Server***           | ***Server***           | ***Server***           |
+| Storage          | ***Storage***          | ***Storage***          | ***Storage***          |
+| Networking       | ***Networking***       | ***Networking***       | ***Networking***       |
+
+Legends for above Comparison table
+- Consumer Managed
+- ***Cloud Provider Managed***
+
+### Problem it addresses
+
+Organizations struggle with effective computing resource utilization in a traditional on-premise set up. The data center has to be built for perceived peak demand. When the workload declines, computing resources would be idle. When the workload spikes beyond the perceived demand, there is shortage of computing resources to process the workload. This lack of scalability leads to increased cost and lack of effective resource utilization.
```

Comment:
Good but make it more tentative. There are likely orgs that are able to effectively manage and utilize on premise resources extremely well. I'd also consider adding something about the advantage of renting compute vs buying it.

Reply:
@JasonMorgan - ok, briefly referenced the advantage of renting(as entire premise of iaas is renting) in 'how it helps' section. would it not be a repetition ? your thoughts, please..

Reply:
Great point, let me re review it now.